pandokia/common.py

```diff
@@ -359,7 +360,7 @@ def expand(text, dictlist=[], valid=None, format=''):
                         val = urlencode(val)
                     result.write(val)
                 elif this_format == 'html':
-                    val = cgi.escape(str(val), quote=True)
```

Comment:
this line was likely never used by pyetc or maybe any of our s/w since the old ver of pandokia had forgotten to import cgi (or we got really lucky calling this function in other modules that DID import cgi)